### PR TITLE
Do not deprecate the autowiring aliases

### PR DESCRIPTION
**Pull request by leofeyer**
Follow-up on #3764

### CODE REVIEW

File: core-bundle/src/Resources/config/services.yml [hunk 1]
```diff
@@ -1039,14 +1046,6 @@ services:
 
     Contao\CoreBundle\Framework\ContaoFrameworkInterface: '@contao.framework'
```
Comment (aschempp) [inline core-bundle/src/Resources/config/services.yml:1040]: This should be deprecated, we don't use that interface. Not sure it was ever available for autowiring?

**Review by aschempp — COMMENTED** [review]
Can you please also add autowiring for the PageRegistry?

File: core-bundle/src/Resources/config/services.yml (hunk 1 above)
Comment (leofeyer) [reply core-bundle/src/Resources/config/services.yml:1040]: Yes, it was.

https://github.com/contao/contao/blob/904fd1846abbb051783b8b6d043abaffab3553bb/core-bundle/src/Resources/config/services.yml#L29-L30

I have just moved it down into the "backwards compatibility" section now.

**Comment by leofeyer** [comment]
> Can you please also add autowiring for the PageRegistry?

Added in 1852f1490272a88cd9df93f9de96a208ba233de6.

**Review by aschempp — APPROVED** [review]

**Pull request merged**